packages/journey-manager/src/index.ts

```diff
@@ -376,12 +326,73 @@ export const run = (props: RunProps): RunOutput => {
     return [];
   };
 
+  /**
+   * UI management
+   */
+
+  // TODO: type this more accurately
+  let uiElement: any;
```

Comment:
```suggestion
  let uiElement: JourneyManagerElement | undefined;
```

Reply:
I couldn't make it work... what the `any` helps with is that it foregoes having to type custom events.

Reply:
Ah ok.